### DISCUSSION TITLE
AG can't load large sessions

### DISCUSSION THREAD

**Discussion by musicgreator** [apohl79/audiogridder#848] [General]
I've noticed, that AG can't load Sessions which require AG to use more than 10GB of RAM (in that range). I noticed this with version 1.1.1 and 1.2.0 Beta 6 (probably all other versions too). When loding a large session, the RAM load climbs up to 15GB or so and and falls back down to below 10 GB again and again. I've sent a bug report a few weeks ago which includes this issue. I`m on windows 10 using Reaper, 64GB of Ram. 

Using the Sandboxing mode in AG solves this, but the CPU load seems too high.

**Comment by Henri75** [comment]
the issue is on the server side I guess ?
The load with sandoxing is really higher ?

**Reply by musicgreator** [reply]
I will do a test with a smaller session, in order to compare both modes. But I noticed  the load was higher than using the plugins natively in the DAW. Needs more testing I guess...

**Comment by Henri75** [comment]
for me this is fake load, related to a buffering issue
